lib/Twig/Extensions/Extension/Date.php

```diff
@@ -31,6 +32,11 @@ class Twig_Extensions_Extension_Date extends Twig_Extension
 
     public function __construct(TranslatorInterface $translator = null)
     {
+        // Ignore the IdentityTranslator, otherwise the parameters won't be replaced properly
+        if ($translator instanceof IdentityTranslator) {
```

Comment:
doesn't this mean this `IdentityTranslator` must be set as requirement in composer of Twig?

Reply:
It's in the same package as the `TranslatorInterface` interface, and same than for the interface, it should work even without.

Reply:
an `instanceof` check does not create a dependency (it creates a dependency for the code wanting to make it `true`)